### PR TITLE
Add progress bar

### LINKED ISSUE #9756: add a progress bar for running the check connection and discover commands
For checking connection and running the discover commands, we should expose a progress bar. Since it's very non-trivial to set this up for the first time, we should have the progress bar behavior match the "expected" durations of each of the operations and exit early.

For checking connections: 15s
For discoveries: 2min

### PR DESCRIPTION
**Pull request by juliachvyrova**
## What
Add Progress Bar for checking connection and running the discover schema

Снимок экрана 2022-02-02 в 01 18 51

If the ProgressBar is full and process has not ended, the message will appear.
Снимок экрана 2022-02-02 в 01 19 53

Closes #9756

### CODE REVIEW

File: airbyte-webapp/src/components/LoadingSchema/LoadingSchema.tsx [hunk 1]
```diff
@@ -19,7 +19,7 @@ const FetchMessage = styled.div`
 
 const LoadingSchema: React.FC = () => (
   <SpinnerBlock>
-    <Spinner />
+    <ProgressBar runTime={120} />
```
Comment (jrhizor) [inline airbyte-webapp/src/components/LoadingSchema/LoadingSchema.tsx:22]: Can we make this 4min when running discover? 2min seems good for check connection.

**Review by jrhizor — APPROVED** [review]
looks good, only one small comment

File: airbyte-webapp/src/components/LoadingSchema/LoadingSchema.tsx [hunk 2]
```diff
@@ -19,7 +19,7 @@ const FetchMessage = styled.div`
 
 const LoadingSchema: React.FC = () => (
   <SpinnerBlock>
-    <Spinner />
+    <ProgressBar runTime={240} />
```
Comment (timroes) [inline airbyte-webapp/src/components/LoadingSchema/LoadingSchema.tsx:22]: Could we please leave an inline comment here in code how we got to this rather arbitrary number? Like that this is an experience based amount of time this task usually runs, but so we'll be able in the future to understand where this number was coming from.

Comment (Jamakase) [reply airbyte-webapp/src/components/LoadingSchema/LoadingSchema.tsx:22]: Hehe, I believe we need to ask @jrhizor for such info.

File: airbyte-webapp/src/locales/en.json [hunk 3]
```diff
@@ -102,6 +102,7 @@
   "form.pkSelected": "{count, plural, =0 { } one {{items}} other {# keys selected}}",
   "form.url.error": "field must be a valid URL",
   "form.setupGuide": "Setup Guide",
+  "form.wait": "Please wait a little bit more . . .",
```
Comment (timroes) [inline airbyte-webapp/src/locales/en.json:105]:
```suggestion
  "form.wait": "Please wait a little bit more …",
```

File: airbyte-webapp/src/components/LoadingSchema/LoadingSchema.tsx (hunk 2 above)
Comment (jrhizor) [reply airbyte-webapp/src/components/LoadingSchema/LoadingSchema.tsx]: This number just is purely a middle-of-the-road estimate based off of personal experiences running checks/discover across docker-compose OSS / kube OSS / cloud. In the future we can tune this based off of run durations when we actually capture them.

File: airbyte-webapp/src/locales/en.json [hunk 4]
```diff
@@ -102,6 +102,7 @@
   "form.pkSelected": "{count, plural, =0 { } one {{items}} other {# keys selected}}",
   "form.url.error": "field must be a valid URL",
   "form.setupGuide": "Setup Guide",
+  "form.wait": "Please wait a little bit more …",
```
Comment (jrhizor) [inline airbyte-webapp/src/locales/en.json:105]:
```suggestion
  "form.wait": "Please wait a little bit more…",
```

**Review by jamakase — APPROVED** [review]

**Pull request merged**